### ISSUE TITLE
Move API interaction commands under an `api` subcommand

### ISSUE THREAD

**Issue by djmitche** [taskcluster/taskcluster-cli#52]
The help currently looks like this:
```
    auth                 Operate on the auth service
    authEvents           Operate on the authEvents service
    awsProvisioner       Operate on the awsProvisioner service
    awsProvisionerEvents Operate on the awsProvisionerEvents service
    config               Get/set taskcluster CLI configuration options
    github               Operate on the github service
    githubEvents         Operate on the githubEvents service
    help                 Prints help for a command.
    hooks                Operate on the hooks service
    index                Operate on the index service
    login                Operate on the login service
    notify               Operate on the notify service
    pulse                Operate on the pulse service
    purgeCache           Operate on the purgeCache service
    purgeCacheEvents     Operate on the purgeCacheEvents service
    queue                Operate on the queue service
    queueEvents          Operate on the queueEvents service
    scheduler            Operate on the scheduler service
    schedulerEvents      Operate on the schedulerEvents service
    secrets              Operate on the secrets service
    signin               Sign-in to get temporary credentials
    treeherderEvents     Operate on the treeherderEvents service
    version              Prints the TaskCluster version.
```
and that makes it hard to spot the few options that aren't about a service.  I'd prefer
```
    api                  Perform TaskCluster API operations
    config               Get/set taskcluster CLI configuration options
    help                 Prints help for a command.
    secrets              Operate on the secrets service
    signin               Sign-in to get temporary credentials
    version              Prints the TaskCluster version.
```

**Comment by jonasfj** [comment]
Lol, you forgot secrets :)

**Comment by djmitche** [comment]
ssshhh, it's secret!

**Comment by t0xicCode** [comment]
we can do this right after #76

**Comment by t0xicCode** [comment]
Closed by #99.
